### DISCUSSION TITLE
Portrait orientation for CRT shaders

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#3261] [Ideas]
Modern screens are much more wide than tall (16:9, 21:9, 32:9), so I wonder if rotating the monitor to portrait will be better: vertical resolution will be increased a lot (e.g. up to 7680), but will the reduced horizontal resolution be sufficient?

HxV
1920x1080 -> 1080x1920
3440x1440 -> 1440x3440
3840x2160 -> 2160x3840
7680x4320 -> 4320x7680
HxV curved
5120x1440 -> 1440x5120
7680x2160 -> 2160x7680

Also, some of the 21:9 and especially 32:9 models are curved... and while CRTs were curved themselves the curve is in the opposite direction. Probably too much work to 'compensate' for that?

**Comment by johnnovak** [comment]
Is this a troll post? 🤣 

Seriously man, I'll leave it to you work out why these things don't make any sense... Just spend 15 mins thinking through the implications... 😄

**Reply by Torinde** [reply]
:) I felt it's not a good idea, but since I'm no expert in that I wanted to get others opinions...

**Reply by johnnovak** [reply]
Just imagine our users seeing the _massive_ letterboxing in portrait mode. There would be riots on the streets 😅 

I'm not even sure you could legitimately call that "letterboxing" anymore 😆
